data/json/mapgen/nested/logger missions nested.json

```diff
@@ -0,0 +1,178 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "occupied_lum_wall_0_0",
```

Comment:
It would probably make sense to combine these into a 2x2 rows definition

Reply:
What do you mean, exactly?

Reply:
The same thing that https://github.com/CleverRaven/Cataclysm-DDA/pull/57483/files/bd6a1d1a0a7c1472e714148411c53010c77d3847#diff-ac40946dd98b497b4e2d24f0c1622f5598074f4ae0ab31a61a8d41cd883ec7c1R5-R10 is doing

Reply:
I am not good with multi-overmap tile definitions, you might want to handle that.

Reply:
Sorry, I didn't realize these were nested, so what I said may not be possible here.